src/relay_protocol.rs

```diff
@@ -0,0 +1,11 @@
+use strum::{EnumIs, FromRepr};
+
+pub const PROTOCOL_IDENTIFIER: u128 = 0x033d77afd0ded498c6ec25c75c5b0cf2;
+
+#[derive(Debug, Copy, Clone, FromRepr, EnumIs)]
+#[repr(u8)]
+pub enum RelayOp {
+    Ping = 0,
+    Stop = 1,
+    Token = 2,
+}
```

Comment:
Don't we also need to responses?

Reply:
All responses are empty messages for now.